firestore/vector_search_distance_threshold.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package firestore
+
+// [START firestore_vector_search_basic]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/firestore"
+)
+
+func vectorSearchDistanceThreshold(w io.Writer, projectID string) error {
+	ctx := context.Background()
+
+	client, err := firestore.NewClient(ctx, projectID)
+	if err != nil {
+		return fmt.Errorf("firestore.NewClient: %w", err)
+	}
+	defer client.Close()
+
+	collection := client.Collection("coffee-beans")
+
+	// Requires a vector index
```

Comment:
nit: provide some more context around what a "vector index" is. Maybe provide a link to documentation on cloud.google.com?

Here and following.

Reply:
Added link to documentation